luatest/assertions.lua

```diff
@@ -411,6 +411,21 @@ function M.assert_items_include(actual, expected, message)
     end
 end
 
+--- Checks that one table does not include any items of another, irrespective of their keys.
+--
+-- @param actual
+-- @param expected
+-- @string[opt] message
+function M.assert_items_exclude(actual, expected, message)
+    if type(actual) ~= 'table' or type(expected) ~= 'table' then
+        failure('Argument 1 and 2 must be tables', nil, 3)
```

Comment:
Should be 2, not 3 (it's the stack level).

Anyway, we don't have such a check in `assert_items_include`. Do we really want it here?

Reply:
True, removed. I must be thought that the message looks strange in case we have non-table argument, but since message in `assert_items_include` is quite similar, it is better to remove this check.